src/lib/component/partial/FilterList/FilterList.js

```diff
@@ -0,0 +1,65 @@
+import React from "/vendor/react";
```

Comment:
This should be a base component IMHO since it doesn't contain any specific bindings to any app state